velox/buffer/Buffer.h

```diff
@@ -133,21 +133,13 @@ class Buffer {
     return pool_;
   }
 
-  bool isMutable() const {
-    return mutable_;
+  virtual bool isMutable() const {
```

Comment:
Yeah, the isView() method always returns false for Buffer, so I don't check it here. But it makes more sense to check !isView() here in case there is another child class of Buffer that override isView().

Reply:
in this case we do not need 
  virtual bool isMutable() const  to be virtual amymore.